config/routes.rb

```diff
@@ -161,6 +161,9 @@
     get '/events/disavow' => 'event_disavowal#new', as: :event_disavowal
     post '/events/disavow' => 'event_disavowal#create', as: :events_disavowal
 
+    get '/users/rules_of_use' => 'users/rules_of_use#new'
+    post '/users/rules_of_use' => 'users/rules_of_use#create'
```

Comment:
sorry I totally misrememberd.... we use the `account/` prefix for these -- that was what I was thinking.

anyways URL doesn't matter that much, just wanted us to find a good home for these